Dockerfile

```
@@ -1,4 +1,4 @@
-FROM python:3.9
+FROM python:3.9.18-bullseye
```

prefer this, to keep security patches up to date:

```suggestion
FROM python:3.9-bullseye
```